common/src/test/java/org/apache/celeborn/common/network/util/TransportConfSuiteJ.java

```diff
@@ -73,6 +73,11 @@ public void testSsltrustStoreReloadIntervalMs() {
     assertEquals(10000, transportConf.sslTrustStoreReloadIntervalMs());
   }
 
+  @Test
+  public void testSslHandshakeTimeoutMs() {
+    assertEquals(10000, transportConf.sslHandshakeTimeoutMs());
```

Comment:
Set and test a different value - not default :)

Reply:
done